packages/mask/background/services/wallet/services/wallet/index.ts

```diff
@@ -320,6 +392,27 @@ export function recoverWalletFromPrivateKey(name: string, privateKey: string) {
     return addWalletFromPrivateKey(ImportSource.UserProvided, name, privateKey)
 }
 
+export async function generateWalletFromKeyStoreJSON(json: string, jsonPassword: string) {
```

Comment:
Align `generateAddressFromMnemonicWords`:

```suggestion
export async function generateAddressFromKeyStoreJSON(json: string, jsonPassword: string) {
```